src/cryptographic_primitives/proofs/low_degree_exponent_interpolation.rs

```diff
@@ -209,9 +207,9 @@ mod tests {
 
         let statement = LdeiStatement::new(&witness, alpha, g, d).unwrap();
 
-        let proof = LdeiProof::prove::<Sha256>(&witness, &statement).expect("failed to prove");
+        let proof = LdeiProof::prove::<H>(&witness, &statement).expect("failed to prove");
```

Comment:
Should we make LdeiProof generic over choice of hash, instead of making its functions generic over `H` (ie. `LdeiProof::<H>::prove(..)` instead of `LdeiProof::prove::<H>(..)`)? Just to match syntax that all other proofs have.

Reply:
Hmm yeah, I guess it also makes sense because the proof is correct only over a specific hash function